.prettierignore

```diff
@@ -1,4 +1,17 @@
 code-examples/protect-page-login/php/composer.lock
 CONTRIBUTING.md
-.github/pull_request_template.md
+.github/*
+tests/*
+src/*
+docs/cli/*
+docs/hydra/cli/*
+docs/keto/cli/*
+docs/kratos/cli/*
+docs/oathkeeper/cli/*
```

Comment:
Good catch to exclude these folders!

You can simplify the syntax a bit if you want:

```suggestion
docs/cli/
docs/hydra/cli/
docs/keto/cli/
docs/kratos/cli/
docs/oathkeeper/cli/
```